### DISCUSSION TITLE
land_cover features overlap, which is unexpected

### DISCUSSION THREAD

**Discussion by JesseCrocker** [OvertureMaps/data#168] [Discussion]
### Category Feedback

I have been examining the land_cover type in 2024-05-16-beta.0, and it looks good, but i have noticed that there is significant overlap between feature types within a zoom range. Within a zoom range, like cartography.min_zoom = 8 and cartography.max_zoom = 15, i would expect features to not overlap because land_cover is generally categorical, it's either forest or grass, not both.

At first I thought this must be an issue of how i was importing the data, but i followed the instructions at https://docs.overturemaps.org/examples/QGIS/ to import landcover into qgis, and saw the issue there.

Qgis screenshot:
<img width="1265" alt="Screenshot 2024-06-07 at 10 26 42 AM" src="https://github.com/OvertureMaps/data/assets/2048291/2750f42e-b70d-4f5d-9683-d685e6dc29b4">

GeoJSON file: https://gist.github.com/JesseCrocker/320d82f746d8f44055c0d73ab21d841b

**Comment by jonahadkins** [comment]
hey @JesseCrocker, we added overlap to the extraction process to prevent slivers, gaps, etc... we've added an additional cartographic property, `sort_key` to the next overture release which defines the recommended feature layering order for land cover. keep in mind these are the raw extractions, there should be some expected processing of the data to suit specific needs.  

from the accepted `sort_key` pr:
<img width="814" alt="Screenshot 2024-06-07 at 12 48 12 PM" src="https://github.com/OvertureMaps/data/assets/5178200/4910b83c-ea84-4c5f-90a5-b05694798874">


happy to help with any other questions!

**Comment by vsharma-next** [comment]
Just encountered this myself. Is the code for translating from ESA world cover to OMF base/landcover publicly available ? 

My interest is not in cartography really but analysis and these overlaps can result in unexpected issues and workarounds. Incidentally, I am curious where the slivers and gaps arise from ? I ask because the base dataset is a raster. My best guess is that you did not want 'blocky' polygons and some smoothing is involved ?  

Would it somehow be possible to separate out the overlaps into separate geometries and denote them as such ? 

But apart from these issues - this dataset is quite useful ! thanks for adding it to the collection 👍

**Reply by jonahadkins** [reply]
Glad it's useful @vsharma-next ! Here's [the write up we did for it,](https://daylightmap.org/2023/10/11/landcover.html) when we released it with Daylight that explains the processing a bit. Happy to answer any other questions.
